COORDINATE_SYSTEM2/COORDINATE_SYSTEM2/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace COORDINATE_SYSTEM2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Point p1 = new Point(3, 8);
+            Point p2 = new Point(9, 4);
+            Point p3 = new Point(5, 10);
+            Point p4 = new Point(1, 1);
+            Point p5 = new Point(7, 0); // додав точки
+            
+            List<Point> listOfPoints = new List<Point>();
```

Comment:
I'd expect to see new `CoordinateSystem coordinateSystem = new CoordinateSystem()` here

Reply:
what do you mean?

Reply:
done